corehq/apps/commtrack/static/commtrack/js/sms.js

```diff
@@ -184,15 +184,21 @@ hqDefine('commtrack/js/sms.js', function () {
             };
         };
     }
-    return {
-        initCommtrackSettingsView: function ($element, settings, other_sms_codes) {
-            var model = new CommtrackSettingsViewModel(other_sms_codes);
-            $element.submit(function () {
-                return model.presubmit();
-            });
 
-            model.load(settings);
-            $element.koApplyBindings(model);
-        }
-    };
+    function initCommtrackSettingsView($element, settings, other_sms_codes) {
+        var model = new CommtrackSettingsViewModel(other_sms_codes);
+        $element.submit(function () {
+            return model.presubmit();
+        });
+
+        model.load(settings);
+        $element.koApplyBindings(model);
+    }
+
+    $(function () {
```

Comment:
This is a fine approach that you've been taking, and it makes it easier to review when there are less things changing. No need to do a pass now if you've got a plan to do it eventually.

(the challenge for me will be to remember that I asked this question here so I don't ask it again in the future—have I asked it before??—if I do, just point me back here 😛)